### PR TITLE
⚠️ Set user agent and build own API client object

### PR DESCRIPTION
**Pull request by dragon-dxw**
Note: this changes how we handle environment variables; take care that this works correctly in staging before deploying to live.

### CODE REVIEW

**Comment by dragon-dxw** [comment]
[Add user-agent to ingester](https://trello.com/c/CzpD33r0/1320-add-user-agent-to-ingester)

File: ds-caselaw-ingester/lambda_function.py [hunk 1]
```diff
@@ -13,14 +13,27 @@
...
 )
+from dotenv import load_dotenv
 from notifications_python_client.notifications import NotificationsAPIClient
 
+load_dotenv()
+
+
 rollbar.init(os.getenv("ROLLBAR_TOKEN"), environment=os.getenv("ROLLBAR_ENV"))
 
+api_client = MarklogicApiClient(
```
Comment (dragon-dxw) [inline ds-caselaw-ingester/lambda_function.py:29]: It's possible we could just drop this environment loading, now I've fixed the tests to not depend on the actual api_client

Comment (anthonyhashemi) [reply ds-caselaw-ingester/lambda_function.py:29]: whats the alternative? we'll still need to read the env vars somehow, right?

Comment (anthonyhashemi) [reply ds-caselaw-ingester/lambda_function.py:29]: ah right

**Review by anthonyhashemi — APPROVED** [review]

**Pull request merged**